core/src/main/java/com/aem/demo/core/models/NewsCardModel.java

```diff
@@ -12,34 +12,30 @@
 @Model(adaptables = SlingHttpServletRequest.class,
         resourceType = NewsCardModel.RESOURCE_TYPE,
         defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
+@Getter
 public class NewsCardModel {
 
     protected static final String RESOURCE_TYPE = "aemtraining/components/content/newscard";
+    public static final String DEFAULT_IMAGE = "https://redzonekickboxing.com/wp-content/uploads/2017/04/default-image.jpg";
```

Comment:
Please always account for the visibility of class members. Don't make public what is enough to be protected, don't make protected what is enough to be package-private, and don't make package-private what can be private)

Concerning these tow constants, `RESOURCE_TYPE` is enogh to be package-private (this is less than protected), dna `DEFAULT_IMAGE` is private by definition)